src/resources/META-INF/plugin.xml

```diff
@@ -53,6 +53,9 @@
            class="org.jetbrains.plugins.d2.action.D2LayoutEngineActionGroup"
            popup="true"
            icon="AllIcons.General.Layout"/>
+    <action id="D2.SketchMode"
+           class="org.jetbrains.plugins.d2.action.D2SketchToggleAction"
+           icon="AllIcons.Actions.EnableNewUi"/>
```

Comment:
Let's use another icon here. Something to highlight is that the style looks hand-drawn. A random option:

![image](https://github.com/develar/d2-intellij-plugin/assets/9569795/3ef4f5db-30e1-4780-bb3e-9fd7138200f2)

Reply:
Could you please provide this icon ID? I'm not sure I've seen it within the available IntelliJ options

Reply:
Sorry, I can't provide you with a specific id. Please, take a look at the existing icons here: https://jetbrains.design/intellij/resources/icons_list/

If there are no suitable icons, consider adding one following the documentation here: https://plugins.jetbrains.com/docs/intellij/icons.html

Reply:
Thank you for your research on that! For me, the pencil looks like the best option here :)

Reply:
Uploaded the first one for now

Reply:
Sorry, which one then?

Option 1 (already uploaded): 
<img width="128" alt="image" src="https://github.com/develar/d2-intellij-plugin/assets/18580361/27683d0c-f2e6-4646-bc44-f64383b26acb">


Option 2:
<img width="128" alt="image" src="https://github.com/develar/d2-intellij-plugin/assets/18580361/1bb01b5f-9bd3-4655-9ef6-2ec46687e533">


Option 3 (yours, has license):
<img width="128" alt="image" src="https://github.com/develar/d2-intellij-plugin/assets/18580361/7abd7110-3aba-4380-af83-884bf3c5a7da">

Reply:
Great. All set then. Thanks for your feedback!